core/sql/sqlcomp/nadefaults.cpp

```diff
@@ -1759,6 +1763,7 @@ SDDkwd__(EXE_DIAGNOSTIC_EVENTS,		"OFF"),
  // exposure.
  DDkwd__(HBASE_DELETE_COSTING,		             "ON"),
  DDflt0_(HBASE_DOP_PARALLEL_SCANNER,             "0."),
+ DDkwd__(HBASE_ESTIMATE_ROW_COUNT_VIA_COPROCESSOR,   "OFF"),
```

Comment:
I can buy that. May I suggest that we leave the old code there for the moment until this can be thoroughly exercised? I can remove the old code in a later check-in.